### DISCUSSION TITLE
XPON ONU STICK SEM POTENCIA

### DISCUSSION THREAD

**Discussion by djrobsonsoares** [Anime4000/RTL960x#204] [Q&A] [answered]
![XPON ORD TX-RX SEM POTENCIA](https://github.com/Anime4000/RTL960x/assets/150203644/865a0299-2464-460c-8df3-887def1bd705)

![XPON ORD TX-RX SEM POTENCIA 03](https://github.com/Anime4000/RTL960x/assets/150203644/75f00201-44d7-4f3d-be2f-9c59123ccb5b)

alguém pode me ajuda não sei o porque o TX  - RX  da minha XPON stick não da potencia já fiz de tudo ?

**Comment by vuducdong** [comment, marked as answer]
you need to:
1, attach fiber optic cable
2, check mac address and mackey

**Reply by djrobsonsoares** [reply]
O   TX -POTENCIA NAO EMITIR SINAL 
O   RX -POTENCIA NAO EMITIR SINAL

SERA QUE ESTA COMPROBLEMA
